src/main/java/danta/core/execution/ContextProcessorEngineImpl.java

```diff
@@ -56,7 +56,7 @@
     @Override
     public List<String> execute(ExecutionContext executionContext, ContentModel contentModel)
             throws AcceptsException, ProcessException {
-        List<String> currentProcessorChain = Collections.synchronizedList(new ArrayList<>());
+        List<String> currentProcessorChain = new ArrayList<>();
```

Comment:
I would undo this change.  It has negligible impact to leave it in place for serial CP execution, but removing it makes it more likely that it will be forgotten in the future when parallel execution is reintroduced.